python/foglamp/tasks/north/sending_process.py

```diff
@@ -374,884 +284,654 @@ def __init__(self, loop=None):
-            # logging from different processes
             SendingProcess._logger.removeHandler(SendingProcess._logger.handle)
-            logger_name = _MODULE_NAME + "_" + str(self.input_stream_id)
-
-            SendingProcess._logger = logger.setup(logger_name, destination=_LOGGER_DESTINATION, level=_LOGGER_LEVEL)
-            
-            # TODO: Create a separate "MicroserviceManagementClient" like client for tasks
-            self._core_task_management_client = MicroserviceManagementClient(self._mgt_address, self._mgt_port)
+            logger_name = _MODULE_NAME + "_" + self._name
+            SendingProcess._logger = logger.setup(logger_name, level=logging.INFO if self._debug_level in [None, 0,
+                                                                                                           1] else logging.DEBUG)
+            _LOGGER = SendingProcess._logger
 
             try:
-                # Set the debug level
-                if self._log_debug_level == 1:
-                    SendingProcess._logger.setLevel(logging.INFO)
-                elif self._log_debug_level >= 2:
-                    SendingProcess._logger.setLevel(logging.DEBUG)
-
-                # Sets the reconfigured logger
-                _LOGGER = SendingProcess._logger
-
-                # Start sending
-                is_started = await self._start(self.input_stream_id)
+                is_started = await self._start()
                 if is_started:
-                    await self.send_data(self.input_stream_id)
-
-                # Stop Sending
+                    await self.send_data()
                 self.stop()
-                SendingProcess._logger.info(_MESSAGES_LIST["i000002"])
+                SendingProcess._logger.info("Execution completed.")
                 sys.exit(0)
-            except Exception as ex:
-                message = _MESSAGES_LIST["e000002"].format(str(ex))
-                SendingProcess._logger.exception(message)
+            except (ValueError, Exception) as ex:
+                SendingProcess._logger.exception(_MESSAGES_LIST["e000002"].format(str(ex)))
                 sys.exit(1)
 
     def stop(self):
-        """ Terminates the sending process and the related plugin
-        Args:
-        Returns:
-        Raises:
-        """
+        """ Terminates the sending process and the related plugin"""
         try:
             self._plugin.plugin_shutdown(self._plugin_handle)
         except Exception:
-            _message = _MESSAGES_LIST["e000007"]
-            SendingProcess._logger.error(_message)
-            self._event_loop.run_until_complete(self._audit.failure(self._AUDIT_CODE, {"error - on stop": _message}))
+            SendingProcess._logger.error(_MESSAGES_LIST["e000007"])
+            self._event_loop.run_until_complete(
+                self._audit.failure(self._AUDIT_CODE, {"error - on stop": _MESSAGES_LIST["e000007"]}))
             raise
+        SendingProcess._logger.debug("Stopped")
```

Comment:
info? as we have ` SendingProcess._logger.info("Started")`

Reply:
done